ktor-http/common/src/io/ktor/http/HeaderValueWithParameters.kt

```diff
@@ -80,15 +80,27 @@ private fun String.checkNeedEscape(): Boolean {
 
     for (index in 0 until length) {
         when (this[index]) {
+            '(',
```

Comment:
Could we extract the constant and write a description(small notes about sources of these symbols)?